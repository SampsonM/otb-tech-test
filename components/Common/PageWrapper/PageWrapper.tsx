import styled from 'styled-components'

const PageWrapper = styled.main`
  padding: 4em;
  background-image: url('images/background.png');
  height: 100vh;
`;

export default PageWrapper
