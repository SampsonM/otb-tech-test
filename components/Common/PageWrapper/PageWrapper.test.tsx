import React from "react"
import { render } from "@testing-library/react"
import PageWrapper from './PageWrapper'

describe("PageWrapper", () => {
	test("PageWrapper fragment matches the snapshot", () => {
		const { asFragment } = render(<PageWrapper />)
		expect(asFragment()).toMatchSnapshot()
	})
})
