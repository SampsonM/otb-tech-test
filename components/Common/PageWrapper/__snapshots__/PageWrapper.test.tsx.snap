// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageWrapper PageWrapper fragment matches the snapshot 1`] = `
<DocumentFragment>
  <main
    class="PageWrapper-otb__sc-5ec17cfc-0 gWybxw"
  />
</DocumentFragment>
`;
